ehcache-xml/src/common/schema/ehcache-core.xsd

```diff
@@ -90,6 +90,7 @@
         <xs:annotation>
           <xs:documentation xml:lang="en">
             The element defines the sizing limits for the default SizeOfEngine.
+            Depricated (for memory units)
```

Comment:
For this element something like this should work:
```

DEPRECATED: Memory units for heap resources, which are the primary use-case for size-of-engines, are deprecated.
```

Reply:
Accommodated - Placed same statement